src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs

```diff
@@ -94,96 +109,108 @@ private static double GetHorizontalScrollWheelDelta(Size size, double delta)
 
 		public ScrollContentPresenter()
 		{
+#if __SKIA__
+			_strategy = CompositorScrollStrategy.Instance;
+#elif __MACOS__
+			_strategy = TransformScrollStrategy.Instance;
+#endif
+
+			_strategy.Initialize(this);
+
 			PointerWheelChanged += ScrollContentPresenter_PointerWheelChanged;
-			LayoutUpdated += ScrollContentPresenter_LayoutUpdated;
 
-			// On Skia, the Scrolling is managed by the ScrollContentPresenter, not the ScrollViewer (as UWP).
+			// On Skia and macOS (as UWP), the Scrolling is managed by the ScrollContentPresenter, not the ScrollViewer.
```

Comment:
I think you've completely changed the sense of this comment... '(as UWP)' should go at the end no? Isn't it ScrollViewer that actually handles the scrolling on UWP?

Reply:
Yeah that's the purpose of this change, my previous comment was misleading. The scrolling is handle by the `ScrollContentPresenter`, i.e. it's the SCP which is clipping it's children and which is effectively applies the offsets (for instance it has all the properties related to scrolling like the [`HorizontalOffset`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.scrollcontentpresenter.horizontaloffset?view=net-5.0) )